snap/snapcraft.yaml

```diff
@@ -112,39 +122,21 @@ parts:
         cp -r jars/* $CRAFT_PART_INSTALL/jars/
         mkdir -p $CRAFT_PART_INSTALL/python
         cp -r python/* $CRAFT_PART_INSTALL/python/
+
+        echo "Installing spark8t"
+        chmod -R 755 $CRAFT_PART_INSTALL/lib/python3.10/site-packages/spark8t/cli/
     override-prime: |
         snapcraftctl prime
         rm -vf usr/lib/jvm/java-11-openjdk-*/lib/security/blacklisted.certs
 
   helper-package:
```

Comment:
**suggestion** Now that we understand snap a bit better, maybe it makes sense to divide the snap into 3 parts:
* spark
* spark8t
* kubectl

I feel these are indeed the components of our snap. @welpaolo: how about that?

Reply:
I think this is a good idea, it will make the structure of the snap much cleaner.

Reply:
Done